src/PostgREST/Config.hs

```diff
@@ -180,11 +180,11 @@ readOptions = do
           (Just "") -> pure Nothing
           aud' -> pure aud'
 
-    optionalListOfText :: C.Key -> C.Parser C.Config [Text]
-    optionalListOfText k =
-      C.optional k (C.list C.string) >>= \case
+    optionalCommaSeparatedList :: C.Key -> C.Parser C.Config [Text]
+    optionalCommaSeparatedList k =
+      C.optional k C.string >>= \case
         Nothing -> pure []
-        Just types -> pure types
+        Just commaSeparatedValues -> pure $ strip <$> splitOn "," commaSeparatedValues
```

Comment:
Down below there's a `splitExtraSearchPath` function which has the same logic. Perhaps that function can be renamed and reused here.

Reply:
I was in doubt rregarding whether refactoring that code would be out of the scope of this PR, will fix. Thanks for the feedback